zipkin-server/src/main/resources/zipkin-server-shared.yml

```
@@ -164,6 +164,7 @@ zipkin:
       health-check:
         enabled: ${ES_HEALTH_CHECK_ENABLED:true}
         interval: ${ES_HEALTH_CHECK_INTERVAL:3s}
+      priority: ${ES_TEMPLATE_PRIORITY:0}
```

to match your docs..

```suggestion
      priority: ${ES_TEMPLATE_PRIORITY:}
```